web/src/components/organisms/Dashboard/index.tsx

```diff
@@ -1,28 +1,128 @@
-import { useAuth } from "@reearth-cms/auth";
+import { PlusOutlined } from "@ant-design/icons";
+import styled from "@emotion/styled";
 import Button from "@reearth-cms/components/atoms/Button";
-import React from "react";
-import { useNavigate } from "react-router-dom";
+import MoleculeHeader from "@reearth-cms/components/molecules/Common/Header";
+import WorkspaceCreationModal from "@reearth-cms/components/molecules/Common/WorkspaceCreationModal";
+import WorkspaceMenu from "@reearth-cms/components/molecules/Common/WorkspaceMenu";
+import Search from "antd/lib/input/Search";
+import Layout, { Content, Header } from "antd/lib/layout/layout";
+import Sider from "antd/lib/layout/Sider";
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
 
 import useHooks from "./hooks";
 
 export type Props = {
   workspaceId?: string;
 };
 
-const Dashboard: React.FC<Props> = ({ workspaceId }) => {
-  const { logout } = useAuth();
-  const navigate = useNavigate();
-
-  const { user } = useHooks(workspaceId);
+const Dashboard: React.FC<Props> = () => {
+  const { workspaceId } = useParams();
+  const [collapsed, setCollapsed] = useState(false);
+  const {
+    user,
+    personalWorkspace,
+    workspaces,
+    currentWorkspace,
+    handleWorkspaceCreate,
+    handleModalClose,
+    handleModalOpen,
+    modalShown,
+  } = useHooks(workspaceId);
 
   return (
     <>
-      <h1>Hello {user.name}</h1>
-      <Button onClick={() => logout()}>logout</Button>
-      <Button onClick={() => navigate("/account")}>Account</Button>
-      <Button onClick={() => navigate("/workspaces")}>Workspace list</Button>
+      <Layout style={{ minHeight: "100vh" }}>
+        <Header>
+          <MoleculeHeader
+            handleModalOpen={handleModalOpen}
+            personalWorkspace={personalWorkspace}
+            workspaces={workspaces}
+            currentWorkspace={currentWorkspace}
+            user={user}
+          ></MoleculeHeader>
+        </Header>
+        <Layout>
+          <Sider
+            collapsible
+            collapsed={collapsed}
+            onCollapse={(value) => setCollapsed(value)}
+            style={{ backgroundColor: "#fff" }}
```

Comment:
For CMS should we use inline styles? Or, even if its just one style, should we use styled components? I feel like we should use styled components to keep all custom styles in the same place.